src/platform/plugins/shared/unified_doc_viewer/public/components/doc_viewer_flyout/doc_viewer_flyout.tsx

```diff
@@ -158,6 +158,13 @@ export function UnifiedDocViewerFlyout({
         return;
       }
 
+      const isResizableButton =
+        (ev.target as HTMLElement).getAttribute('data-test-subj') === 'euiResizableButton';
+      if (isResizableButton) {
+        // ignore events triggered when the resizable button is focused
+        return;
+      }
```

Comment:
It feels a bit like playing whack-a-mole with all these conditions, but I don't have a better suggestion, so what's one more I suppose 😅